views/shared/authenticated_user_navbar.html

```diff
@@ -0,0 +1,32 @@
+<nav class="navbar fixed-top navbar-expand-lg navbar-dark bg-dark">
+  <a class="navbar-brand" href="/">Google Scraper App</a>
+  <button
+    class="navbar-toggler"
+    type="button"
+    data-toggle="collapse"
+    data-target="#navbarSupportedContent"
+    aria-controls="navbarSupportedContent"
+    aria-expanded="false"
+    aria-label="Toggle navigation"
+  >
+    <span class="navbar-toggler-icon"></span>
+  </button>
+
+  <div class="collapse navbar-collapse" id="navbarSupportedContent">
+    <ul class="navbar-nav mr-auto">
+      <li class="nav-item active">
+        <a class="nav-link" href="/">Dashboard</a>
+      </li>
+    </ul>
+
+    <ul class="navbar-nav ml-auto">
+      <li class="nav-item">
+        <span class="navbar-text">Logged in as {{ .CurrentUser.Name }}</span>
+      </li>
+
+      <li class="nav-item active">
```

Comment:
@junan Not sure why does this have to include the `active` class 🤔

Reply:
`active` class makes the link text bright, which looks good to me. Thats why I added the class

Reply:
@junan I mean normally we use `active` class on an active nav item so using it here just to style the link looks a bit weird to me 🤔  How about using another class here and set the color to that class?